nservicebus/transports/guidance.md

```diff
@@ -0,0 +1,114 @@
+---
+title: SQL Server Transport usage guidance
+summary: Usage guidance for SQL Server transport
+tags:
+- SQL Server
+---
+
+The SQL Server transport implements a message queueing mechanism on top of Microsoft SQL Server. It provides support for sending messages over [SQL Server](http://www.microsoft.com/en-au/server-cloud/products/sql-server/) tables. It does **not** make any use of ServiceBroker, a messaging technology built into the SQL Server.
+
+
+## How it works
+
+The SQL Server transport is a hybrid queueing system which is neither store-and-forward (like MSMQ) nor a broker (like RabbitMQ). It treats the SQL Server as a storage infrastructure for the queues while the queue implementation logic is executed inside the client process.
+
+
+### Advantages of SQL Server Transport
+
+ * No additional licensing and training costs, as majority of Microsoft stack organizations already have SQL Server installed and have knowledge required to run it
+ * Great general-purpose tooling (SSMS)
+ * Throughput on par with MSMQ
+ * Scaling out of processing is simple because of support for competing consumers at the queue level
+
+## Deployment considerations
+
+### Introduction
+Any NServiceBus endpoint operates and manages different types of data, those are:
+ * Business data - data used for implementing business capabilities
+ * Persistence data - infrastructure level data including: saga data, timeout manager state and message driven pub/sub information
+ * Transport data - messaging infrastructure state
+
+SQL Server Transport manages transport data and puts no constraints on the type and configuration of storage technology used for persistence and business data. It can work with any of available persisters i.e. NHibernate and RavenDB, as well as any storage mechanisms used inside message handlers. However, sharing storage technology (SQL Server) between transport, persistence and/or business data offers unique advantages (see next section) and, as a result, this guidance will focus on such scenario.
+
+From data perspective, deployment model chosen by system architect answers the question of how business, persistence and transport data is stored in terms of SQL Server instances, catalogs and schemas.
+
+NOTE: No matter which deployment option is chosen, there is one general rule that should always apply: **All transport data (input, audit and error queues) should reside on a single SQL Server catalog**.  
+
+### Transactionality (Exactly once message delivery)
+When sharing single technology for different types of data it is possible to provide `exactly-once` message processing without need for DTC. In all other cases it is required to turn on DTC.
+
+#### Business, Persistence and Transport data in single catalog
+Enables `exactly-once` message delivery with native transactions. Multi-catalog with local transactions are possible in principle but currently not supported by SQL Server transport. There is possibility to use different schemas.
```

Comment:
Multi-catalog is not defined about so readers would not know what does it mean in this context.